Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG PYTHON_VERSION=3.8
-FROM python:${PYTHON_VERSION}-slim AS application
+ARG PYTHON_VERSION=3.8.11
+FROM python:${PYTHON_VERSION}-slim-bullseye AS application
```

Comment:
It's also good to define which version of the OS we're using. Currently, the Python version can be with either `buster` or `bullseye`.